omod/src/main/webapp/resources/scripts/metadata/manageRoles.js

```diff
@@ -0,0 +1,283 @@
+angular.module("manageRoles", [ "roleService", "privilegeService", "ngDialog", "ui.router", "uicommons.filters", "colTable" ])
```

Comment:
Why is this creeping into this? I think you didn't create your branches quite well